src/widgets/splits/Split.hpp

```diff
@@ -3,6 +3,7 @@
 #include "common/Aliases.hpp"
 #include "common/Channel.hpp"
 #include "common/NullablePtr.hpp"
+#include "messages/MessageThread.hpp"
```

Comment:
Can be moved to `Split.cpp` and replaced with a forward declaration `class MessageThread;`.
```suggestion
```